<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PbxForm</class>
 <widget class="QWidget" name="PbxForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
border-style: solid;
border-width: 1px;
border-radius: 10px;
border-color: grey;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Call Me Now</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,3,3,3,0,0,1">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="userPhoneLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="callUserButton">
          <property name="text">
           <string>Call User</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="agentPhoneLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="callAgentButton">
          <property name="text">
           <string>Call Agent</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="thirdPhoneLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="callThirdButton">
          <property name="text">
           <string>Call Third</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="bridgeStatusLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>FreeMono</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>User-Agent</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bridgeButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Bridge</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="redirectButton">
        <property name="text">
         <string>Redirect third</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hangupAllButton">
        <property name="text">
         <string>Hang up all channels</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Twilio Mockup</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,0,0,0,5">
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="inboundPhoneLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="inboundButton">
          <property name="text">
           <string>Inbound Call</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="ivPhoneLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="ivButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>IV Call</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="pinOKCheckBox">
        <property name="text">
         <string>Reply OK to Check IV pin</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="hangupPhoneLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="hangupButton">
          <property name="text">
           <string>Hangup Call</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
